tensorflow/g3doc/get_started/os_setup.md

```diff
@@ -61,7 +61,17 @@ $ sudo easy_install pip
 $ sudo easy_install --upgrade six
 ```
 
-Then, select the correct binary to install:
+We have also uploaded the CPU version of the binaries to Pypi, so you can
+simply install on Linux, Mac or Windows with:
+
+```bash
+$ pip install tensorflow
```

Comment:
Gpu versions are too big to upload to pypi. So at the moment, we only have CPU in pypi.
I think this should be mentioned.